Failed test at my locale on master since f815b58d

On master branch at my locale

```console
make test TEST=tests/test_markup.py
```

has one failing test. Bisecting leads to f815b58d.

Here is console output with a checkout of this commit.

```console
============================= test session starts ==============================
platform darwin -- Python 3.6.6, pytest-3.9.3, py-1.7.0, pluggy-0.8.0 -- /Users/xxxxxx/Downloads/sphinxvenv/bin/python3
cachedir: .pytest_cache
libraries: Sphinx-2.0.0+/f815b58df, docutils-0.14
rootdir: /Users/xxxxxx/sphinx, inifile: setup.cfg
collected 19 items                                                             

tests/test_markup.py::test_inline[verify_re-``code   sample``-<p><code class="(samp )?docutils literal notranslate"><span class="pre">code</span>&#160;&#160; <span class="pre">sample</span></code></p>-\\\\sphinxcode{\\\\sphinxupquote{code   sample}}] FAILED [  5%]
[... cut lines ...]
tests/test_markup.py::test_default_role2 PASSED                          [100%]

=================================== FAILURES ===================================
 test_inline[verify_re-``code   sample``-<p><code class="(samp )?docutils literal notranslate"><span class="pre">code</span>&#160;&#160; <span class="pre">sample</span></code></p>-\\\\sphinxcode{\\\\sphinxupquote{code   sample}}] 

get_verifier = <function get_verifier.<locals>.get at 0x106c04268>
type = 'verify_re', rst = '``code   sample``'
html_expected = '<p><code class="(samp )?docutils literal notranslate"><span class="pre">code</span>&#160;&#160; <span class="pre">sample</span></code></p>'
latex_expected = '\\\\sphinxcode{\\\\sphinxupquote{code   sample}}'

    @pytest.mark.parametrize('type,rst,html_expected,latex_expected', [
        (
            # correct interpretation of code with whitespace
            'verify_re',
            '``code   sample``',
            ('<p><code class="(samp )?docutils literal notranslate"><span class="pre">'
             'code</span>&#160;&#160; <span class="pre">sample</span></code></p>'),
            r'\\sphinxcode{\\sphinxupquote{code   sample}}',
        ),
        (
            # correct interpretation of code with whitespace
            'verify_re',
            ':samp:`code   sample`',
            ('<p><code class="(samp )?docutils literal notranslate"><span class="pre">'
             'code</span>&#160;&#160; <span class="pre">sample</span></code></p>'),
            r'\\sphinxcode{\\sphinxupquote{code   sample}}',
        ),
        (
            # interpolation of braces in samp and file roles (HTML only)
            'verify',
            ':samp:`a{b}c`',
            ('<p><code class="samp docutils literal notranslate">'
             '<span class="pre">a</span>'
             '<em><span class="pre">b</span></em>'
             '<span class="pre">c</span></code></p>'),
            '\\sphinxcode{\\sphinxupquote{a\\sphinxstyleemphasis{b}c}}',
        ),
        (
            # interpolation of arrows in menuselection
            'verify',
            ':menuselection:`a --> b`',
            ('<p><span class="menuselection">a \N{TRIANGULAR BULLET} b</span></p>'),
            '\\sphinxmenuselection{a \\(\\rightarrow\\) b}',
        ),
        (
            # interpolation of ampersands in menuselection
            'verify',
            ':menuselection:`&Foo -&&- &Bar`',
            ('<p><span class="menuselection"><span class="accelerator">F</span>oo '
             '-&amp;- <span class="accelerator">B</span>ar</span></p>'),
            r'\sphinxmenuselection{\sphinxaccelerator{F}oo -\&- \sphinxaccelerator{B}ar}',
        ),
        (
            # interpolation of ampersands in guilabel
            'verify',
            ':guilabel:`&Foo -&&- &Bar`',
            ('<p><span class="guilabel"><span class="accelerator">F</span>oo '
             '-&amp;- <span class="accelerator">B</span>ar</span></p>'),
            r'\sphinxguilabel{\sphinxaccelerator{F}oo -\&- \sphinxaccelerator{B}ar}',
        ),
        (
            # non-interpolation of dashes in option role
            'verify_re',
            ':option:`--with-option`',
            ('<p><code( class="xref std std-option docutils literal notranslate")?>'
             '<span class="pre">--with-option</span></code></p>$'),
            r'\\sphinxcode{\\sphinxupquote{-{-}with-option}}$',
        ),
        (
            # verify smarty-pants quotes
            'verify',
            '"John"',
            '<p>“John”</p>',
            "“John”",
        ),
        (
            # ... but not in literal text
            'verify',
            '``"John"``',
            ('<p><code class="docutils literal notranslate"><span class="pre">'
             '&quot;John&quot;</span></code></p>'),
            '\\sphinxcode{\\sphinxupquote{"John"}}',
        ),
        (
            # verify classes for inline roles
            'verify',
            ':manpage:`mp(1)`',
            '<p><em class="manpage">mp(1)</em></p>',
            '\\sphinxstyleliteralemphasis{\\sphinxupquote{mp(1)}}',
        ),
        (
            # correct escaping in normal mode
            'verify',
            'Γ\\\\∞$',
            None,
            'Γ\\textbackslash{}\\(\\infty\\)\\$',
        ),
        (
            # in verbatim code fragments
            'verify',
            '::\n\n @Γ\\∞${}',
            None,
            ('\\begin{sphinxVerbatim}[commandchars=\\\\\\{\\}]\n'
             '@Γ\\PYGZbs{}\\(\\infty\\)\\PYGZdl{}\\PYGZob{}\\PYGZcb{}\n'
             '\\end{sphinxVerbatim}'),
        ),
        (
            # in URIs
            'verify_re',
            '`test <http://example.com/~me/>`_',
            None,
            r'\\sphinxhref{http://example.com/~me/}{test}.*',
        ),
    ])
    def test_inline(get_verifier, type, rst, html_expected, latex_expected):
        verifier = get_verifier(type)
>       verifier(rst, html_expected, latex_expected)

tests/test_markup.py:240: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_markup.py:108: in verify_re_
    verify_re_latex(rst, latex_expected)
tests/test_markup.py:93: in verify
    app.builder.init_context()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <sphinx.builders.latex.LaTeXBuilder object at 0x106c00da0>

    def init_context(self):
        # type: () -> None
        self.context = DEFAULT_SETTINGS.copy()
    
        # Add special settings for latex_engine
        self.context.update(ADDITIONAL_SETTINGS.get(self.config.latex_engine, {}))
    
        # for xelatex+French, don't use polyglossia by default
        if self.config.latex_engine == 'xelatex':
            if self.config.language:
                if self.config.language[:2] == 'fr':
                    self.context['polyglossia'] = ''
                    self.context['babel'] = r'\usepackage{babel}'
    
        # Apply extension settings to context
>       self.context['packages'] = self.usepackages
E       AttributeError: 'LaTeXBuilder' object has no attribute 'usepackages'

sphinx/builders/latex/__init__.py:191: AttributeError
--------------------------- Captured stdout teardown ---------------------------
# testroot: root
# builder: latex
# srcdir: /Users/xxxxxx/sphinx/tests/build/root
# outdir: /Users/xxxxxx/sphinx/tests/build/root/_build/html
# status: 
Running Sphinx v2.0.0+/f815b58df

# warning: 

=============================== warnings summary ===============================
[... cut lines ...]
=============== 1 failed, 18 passed, 10 warnings in 0.93 seconds ===============
make: *** [Makefile:68: test] Error 1
(sphinxvenv) 
```

Same error is raised on master branch (at 74e16bcb).

```console
>       self.context['packages'] = self.usepackages
E       AttributeError: 'LaTeXBuilder' object has no attribute 'usepackages'

sphinx/builders/latex/__init__.py:194: AttributeError
--------------------------- Captured stdout teardown ---------------------------
# testroot: root
# builder: latex
# srcdir: /Users/xxxxxx/sphinx/tests/build/root
# outdir: /Users/xxxxxx/sphinx/tests/build/root/_build/html
# status: 
Running Sphinx v2.0.0+/74e16bcb7

# warning: 

===================== 1 failed, 21 passed in 1.01 seconds ======================
make: *** [Makefile:68: test] Error 1
(sphinxvenv) 
```

 However when running `make test` no failed test is reported.

```console
tests/test_versioning.py::test_insert_similar PASSED                     [ 99%]
tests/test_writer_latex.py::test_rstdim_to_latexdim PASSED               [100%]

================== 1367 passed, 10 skipped in 134.10 seconds ===================
(sphinxvenv) 
```

--- a/tests/test_markup.py
+++ b/tests/test_markup.py
@@ -90,8 +90,7 @@ def verify_re_latex(app, parse):
         document = parse(rst)
         app.builder = LaTeXBuilder(app)
         app.builder.set_environment(app.env)
-        app.builder.init_context()
-        app.builder.init_babel()
+        app.builder.init()
         latex_translator = ForgivingLaTeXTranslator(document, app.builder)
         latex_translator.first_document = -1  # don't write \begin{document}
         document.walkabout(latex_translator)

